symengine/polys/uintpoly_piranha.h

```diff
@@ -70,6 +108,36 @@ class UIntPolyPiranha : public UIntPolyBase<pintpoly, UIntPolyPiranha>
         return poly_.degree();
     }
 
+    // begin() and end() are unordered
+    // rbegin() and rend() are ordered
```

Comment:
Because the standard name implies a reverse iteration. This is not true for `UIntPolyPiranha`.